Add Custom PrecenseFilter to LDAP Auth (#6208)
solo-io/gloo: docs/content/guides/security/auth/extauth/ldap/_index.md

- `membershipAttributeName`: case-insensitive name of the attribute that contains the names of the groups an entry is 
   member of. Defaults to `memberOf` if not provided.
- `allowedGroups`: DNs of the user groups that are allowed to access the secured upstream.

> - `searchFilter`: The filter to use when searching for a member to, then, authorize.

Reviewer: I wasn't sure what the "then" meant; is there a next step? Is it a filter for a member or user entry (maybe they're the same thing?)
```suggestion
- `searchFilter`: The filter to use when searching for the user entry that you want to authorize.
```
